bridge/svix-bridge/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2021-2023 Svix Inc.
```

Comment:
No need to have a license file in every directory (I think?), just have one top-level for Svix-bridge.

Reply:
I started out with one top-level license, but the security linter called these crates errors without the license in each. I think we need them in here.